platforms/chibios/bootloader.c

```diff
@@ -123,10 +123,16 @@ void enter_bootloader_mode_if_requested(void) { /* Jumping to bootloader is not
 #    if defined(BOOTLOADER_KIIBOHD)
 /* Kiibohd Bootloader (MCHCK and Infinity KB) */
 #        define SCB_AIRCR_VECTKEY_WRITEMAGIC 0x05FA0000
+
+// taken from Kiibohd/controller
+#        define VBAT_SECURE1            *(volatile uint32_t *)0x4003E018 // Kiibohd Secure 32 bit register 1 (24th byte)
+#        define VBAT_SECURE2            *(volatile uint32_t *)0x4003E01C // Kiibohd Secure 32 bit register 2 (28th byte)
 const uint8_t              sys_reset_to_loader_magic[] = "\xff\x00\x7fRESET TO LOADER\x7f\x00\xff";
```

Comment:
```suggestion
const uint8_t              sys_reset_to_loader_magic[] = "\xFF\x00\x7FRESET TO LOADER\x7F\x00\xFF\x00\x00\x00\x00\x00\x00\x00\x00";
```

Reply:
If this doesn't work, it might need three more `\x00`s on the end to fill out the 32 bytes.

Reply:
I added enough to fill out 31 bytes, and with the string terminator it is 32 bytes. Anything less or more doesn't work.